sdk/servicebus/Azure.ResourceManager.ServiceBus/tests/Tests/ServiceBusNamespaceTests.cs

```diff
@@ -173,8 +173,13 @@ public async Task GetPrivateLinkResources()
             ServiceBusNamespace serviceBusNamespace = (await namespaceCollection.CreateOrUpdateAsync(namespaceName, new ServiceBusNamespaceData(DefaultLocation))).Value;
 
             //get private link resource
-            IReadOnlyList<PrivateLinkResource> privateLinkResources = (await serviceBusNamespace.GetPrivateLinkResourcesAsync()).Value;
-            Assert.NotNull(privateLinkResources);
+            //get private link resource
+            await foreach (var _ in serviceBusNamespace.GetPrivateLinkResourcesAsync())
+            {
+                return;
+            }
+
+            Assert.Fail($"{nameof(ServiceBusNamespace)}.{nameof(ServiceBusNamespace.GetPrivateLinkResourcesAsync)} has returned an empty collection of {nameof(PrivateLinkResource)}.");
```

Comment:
I think we should return this.  Since we expect an potentially empty list we only want to assert getting the list works.  If getting the list has an issue it will be a RequestFailedException and the test will fail due to that.  Therefore we simply need to call the foreach to iterate and not assert anything afterwards

Reply:
Fixed